src/LEDMatrixDriver.hpp

```diff
@@ -103,10 +111,11 @@ class LEDMatrixDriver
 		uint8_t* _getBufferPtr(int16_t x, int16_t y) const;
 		void _sendCommand(uint16_t command);
 		void _displayRow(uint8_t row);
+		uint8_t _reverseBits(uint8_t n);
 
 		const uint8_t N;
 		SPISettings spiSettings;
-		bool modRev = false;
+		uint8_t moduleFlip;

```

Comment:
Now the type internally is the raw one... Ugh, but there's no nice way to handle that I guess.